test-setup.js

```diff
@@ -1,10 +1,64 @@
 /* eslint-disable global-require,no-plusplus */
-import 'babel-polyfill';
+import '@babel/polyfill';
 import 'isomorphic-fetch';
 import 'raf/polyfill';
+import { Headers } from 'node-fetch';
 import { configure } from 'enzyme';
 import dateMock from './mocks/dateMock';
 
+/**
+ * lets mock i18next for all tests
+ */
+jest.mock('i18next', () => {
+	const noop = () => {};
+	const i18n = {
+		t: (msg, options) => {
+			let buff = options.defaultValue || msg;
+			const split = buff.split('}}');
+			if (split.length > 1) {
+				buff = split.reduce((acc, current) => {
+					const sub = current.split('{{');
+					let value = sub.length > 1 ? options[sub[1]] : '';
+					if (value === undefined) {
+						value = '';
+					}
+					return `${acc}${sub[0]}${value}`;
+				}, '');
+			}
+			return buff;
```

Comment:
I would love to extract out this function but jest.mock do not let me doing so, the code is hoisted by jest.